auth-server/src/utils/jwtHelpers.js

```diff
@@ -31,10 +31,20 @@ function shouldUpdateJWT(token, email, role) {
     if (String(userEmail) === String(email) && userRole === role) {
       return false;
     }
-    return false;
+    return true;
   } catch (err) {
     return err;
   }
+  // } catch (err) {
```

Comment:
If you dont think you'll reference this commented code in the near future, go ahead and remove it to keep the codebase clean